README.md

```diff
@@ -67,3 +67,23 @@ Project.on('error', function(msg){
 	// msg.detail -- 'some helpful error comment'
 });
 ```
+
+### Set data to extraData object
+
+`.setState()` attaches an additional key to this object with the key and value passed.
+
+Example:
+
+```javascript
+Project.setState('companyID', 1);
+```
+
+### Get data in the extraData object
+
+`getState()` will return all the data that is currently set in `this.extraData`;
+
+Example:
+
+```javascript
+const extraData = this.getState();
```

Comment:
Can you add an example of the return value structure here? Something like:
```javascript
==>
{
  "key1" : "value1",
  "key2" : "value2",
  ...
  "keyN" : "valueN"
}

Reply:
Ngl, i was struggling how to display it :) 
Will add this to the README.